sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -304,6 +306,25 @@ public List<SampleDto> getByLabSampleId(String labSampleId) {
 		return em.createQuery(cq).getResultList().stream().distinct().map(SampleFacadeEjb::toDto).collect(Collectors.toList());
 	}
 
+	@Override
+	@PermitAll
```

Comment:
shouldn't we use a narrower range of permissions here? We only need Case, Contact and Event Participant view.

Reply:
Good point, will change it accordingly